.npmignore

```diff
@@ -1,14 +1,9 @@
 /.github/
 /.psci*
 /.pulp-cache
-/binaries
-/bower_components/
-/build.sh
-/deploy.sh
```

Comment:
I have a `deploy.sh` script locally that I use to upload the latest version to https://insect.sh/. It's not being tracked in Git. I could move that somewhere else locally, but the easiest option for now would be to leave this entry in `.npmignore`.

Reply:
Ah, okay, I thought it was something that was removed in a previous commit. How about adding a comment to clarify?

Reply:
That would be great.

I can optionally also check in that script (basically just an rsync call) but it will obviously only work for me

Reply:
That's possible, but I think we should stick to a comment instead.